src/style.css

```diff
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.search-bar {
+  margin: 20px 0;
+  position: relative;
+}
+
+.search-input {
+  box-sizing: border-box;
+  width: 100%;
+  height: 42px;
+  padding: 0 33px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font: inherit;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.big .search-input {
+  font-size: 16px;
+  height: 70px;
+  padding: 0 62px;
+}
+
+.search-button {
+  position: absolute;
+  top: 14px;
+  left: 13px;
+  margin: 0;
+  padding: 0;
+  border: none;
+  background: none;
+}
+
+.big .search-button {
+  top: 25px;
+  left: 26px;
+}
```

Comment:
nice work with the selectors here 👍 
still there's a better way of doing it)

you can make one selector for the searchbar and add the repeated rules there
```
.searchBar {
** all repeated styling + styling for big searchbar goes here **
}

.searchBar--small {
** here goes styling that differs **
}
```